platform/persistence/platform-persistence-commands/src/main/java/org/codice/ddf/persistence/commands/AbstractStoreCommand.java

```diff
@@ -27,7 +29,7 @@
  * Abstract store command that allows store commands to be built off this. Takes care of obtaining
  * the persistent store service, console, and logging.
  */
-public abstract class AbstractStoreCommand extends OsgiCommandSupport {
+public abstract class AbstractStoreCommand implements Action {
 
     protected final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
@josephthweatt did you forget to push up this change? That is the only thing I see before building it